modules/installation/pages/install-vm.adoc

```diff
@@ -29,30 +23,32 @@ In the following table replace _version_ with the actual product version string.
 | Version:            | SLES_version_-JeOS-for-kvm-and-xen.x86_64-GM.qcow2
 | Memory:             | 4096 MB
 | CPU's:              | 2
-| Storage Format:     |_.qcow2_ 24 GB (Default) JeOS Root Partition
+| Storage Format:     | _.qcow2_ 24 GB (Default) JeOS Root Partition
 | Virtual Disks:      |
 | VirtIO Disk 2       | 101 GB for [path]``/var/spacewalk``
 | VirtIO Disk 3       | 50 GB for [path]``/var/lib/pgsql``
 | VirtIO Disk 4       | 4 GB for swap
 | Name:               | test-setup
-| Network             |Bridge _br0_
+| Network             | Bridge _br0_
 |===
 
-.{suse} Virtualization Guide
+.{sle} Virtualization Guide
 [TIP]
```

Comment:
I don't think this needs to be in an admonition. Plus, we don't use the [TIP] admonition.